app/_locales/en/messages.json

```
@@ -1600,6 +1600,9 @@
   "transactionTime": {
     "message": "Transaction Time"
   },
+  "showTransactionTimeDescription": {
+    "message": "Select this to display pending transaction time estimates in the activity tab. Note: estimates are approximations based on network conditions."
```

Perhaps we should specify here that it's specific to mainnet as well?
```suggestion
    "message": "Select this to display pending transaction time estimates in the activity tab while on the Main Ethereum Network. Note: estimates are approximations based on network conditions."
```

---

Thanks!